pkg/operator/cluster/operator_test.go

```diff
@@ -117,4 +123,48 @@ var _ = Describe("Cluster Operator", func() {
 				}))
 		})
 	})
+
+	When("a cluster is updated", func() {
+		var (
+			updatedCluster *crv1.MySQLCluster
+		)
+
+		BeforeEach(func() {
+			updatedCluster = new(crv1.MySQLCluster)
+			updatedCluster = cluster.DeepCopy()
+			updatedCluster.Spec.Port = updatedCluster.Spec.Port + 1
+			updatedCluster.Spec.Replicas = updatedCluster.Spec.Replicas + 1
```

Comment:
The tests looks fine, but I think it would be better to split this into two separate tests (one for `Port` and one for `Replicas`).